WooCommerce/Classes/Extensions/Decimal+Helpers.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension Decimal {
+
+    /// Returns the int value of a decimal.
```

Comment:
Please describe how the fraction is treated in the comment too (up/down rounding)